app/models/validations/shared_validations.rb

```diff
@@ -19,17 +19,19 @@ def validate_numeric_min_max(record)
       next unless record[question.id]
 
       field = question.check_answer_label || question.id
+      min = [question.prefix, question.min].join("")
```

Comment:
Nice, although it might be good to also add a comma delimiter between the 1000s?

```suggestion
      min = [question.prefix, number_with_delimiter(question.min, :delimiter => ',')].join("")
```

Reply:
Nice one!